src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -183,19 +177,21 @@ void GetZoneEquipment(EnergyPlusData &state)
     // PURPOSE OF THIS SUBROUTINE:
     // Get all the system related equipment which may be attached to a zone
 
-    if (!state.dataZoneEquip->ZoneEquipInputsFilled) {
+    if (state.dataZoneEquipmentManager->GetZoneEquipmentInputFlag) {
         GetZoneEquipmentData(state);
-    }
+        state.dataZoneEquipmentManager->GetZoneEquipmentInputFlag = false;
+        state.dataZoneEquip->ZoneEquipInputsFilled = true;
 
-    state.dataZoneEquipmentManager->NumOfTimeStepInDay = state.dataGlobal->NumOfTimeStepInHour * 24;
+        state.dataZoneEquipmentManager->NumOfTimeStepInDay = state.dataGlobal->NumOfTimeStepInHour * 24;
 
-    int MaxNumOfEquipTypes = 0;
-    for (int Counter = 1; Counter <= state.dataGlobal->NumOfZones; ++Counter) {
-        if (!state.dataZoneEquip->ZoneEquipConfig(Counter).IsControlled) continue;
-        MaxNumOfEquipTypes = max(MaxNumOfEquipTypes, state.dataZoneEquip->ZoneEquipList(Counter).NumOfEquipTypes);
-    }
+        int MaxNumOfEquipTypes = 0;
+        for (int Counter = 1; Counter <= state.dataGlobal->NumOfZones; ++Counter) {
+            if (!state.dataZoneEquip->ZoneEquipConfig(Counter).IsControlled) continue;
+            MaxNumOfEquipTypes = max(MaxNumOfEquipTypes, state.dataZoneEquip->ZoneEquipList(Counter).NumOfEquipTypes);
+        }
 
-    state.dataZoneEquipmentManager->PrioritySimOrder.allocate(MaxNumOfEquipTypes);
+        state.dataZoneEquipmentManager->PrioritySimOrder.allocate(MaxNumOfEquipTypes);
```

Comment:
That's it?  That's the whole change here?  OK, love the small amount of code to review!